csv_feed:
  ros__parameters:
    csv_path: "/home/thijs/magcsv/rosbag2_2021-12-09_15-05-39_2xRectangleRounded_0.csv"
    interval: 0.025
    train_ratio: 0.55
    sensor_frame: "xsens_link"
    world_frame: "world"
    mag_topic: "imu/mag"
